open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-api/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/ffdc/AnalyticsModelingErrorCode.java

```diff
@@ -164,7 +164,16 @@
 			"User {0} failed to create server capability {1}. Cause: {2}",
 			"The system is unable to process the request.",
 			"Review the exception to resolve the issue."),
-	
+	FAILED_UPDATE_ARTIFACT(
+			"OMAS-ANALYTICS-MODELING-027", 
+			"Failed to update analytics artifact.",
```

Comment:
The exception is not provided in the message

Reply:
fixed